packages/prescriptionListLambda/tests/test_prescriptionsLookupService.test.ts

```diff
@@ -0,0 +1,247 @@
+import {
+  jest,
+  describe,
+  it,
+  expect,
+  beforeEach
+} from "@jest/globals"
+import {getPrescriptions, PrescriptionError} from "../src/services/prescriptionsLookupService"
+import {Logger} from "@aws-lambda-powertools/logger"
+import {AxiosInstance, AxiosResponse} from "axios"
+import {Bundle} from "fhir/r4"
+import {TreatmentType} from "../src/types"
+
+const mockGet = jest.fn(() => Promise.resolve({} as unknown as AxiosResponse))
+const mockAxiosInstance = {
+  get: mockGet
+}
+
+jest.mock("axios", () => ({
+  create: () => mockAxiosInstance
+}))
+
+describe("Prescriptions Lookup Service Tests", () => {
+  const mockLogger = new Logger({serviceName: "test"})
```

Comment:
Similar suggestion here - pull the mock objects out to another file. Can this re-use the other file's data?